### PR TITLE
adding user info and help page setup

### PR DESCRIPTION
**Pull request by Sahil590**
# Description
Added the current user login info in the Navar.
Initial setup of the help page.
 

Fixes # (issue)
#91 
#96 
## Type of change

- [ ] Documentation (non-breaking change that adds or improves the documentation)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (non-breaking, back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (whatever its nature)

## Key checklist

- [x] All tests pass (eg. `python -m pytest`)
- [x] The documentation builds and looks OK (eg. `python -m sphinx -b html docs docs/build`)
- [x] Pre-commit hooks run successfully (eg. `pre-commit run --all-files`)

## Further checks

- [x] Code is commented, particularly in hard-to-understand areas
- [x] Tests added or an issue has been opened to tackle that in the future. (Indicate issue here: # (issue))

### CODE REVIEW

File: main/templates/main/help.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{% extends "main/base.html" %}
+{% block title %}
+  Help
+{% endblock title %}
```
Comment (dalonsoa) [inline main/templates/main/help.html:4]: I've seen it in other pages - I think the process manager and controller do the same - but why do we have a title block if we are then adding a `h1` header within the content block?

**Review by dalonsoa — APPROVED** [review]
LGTM!

File: main/templates/main/help.html (hunk 1 above)
Comment (Sahil590) [reply main/templates/main/help.html:4]: This was just something I was playing around with, wanted to see the difference between the two.

Comment (cc-a) [reply main/templates/main/help.html]: The title block doesn't add content to the page it sets the content of the [title tag](https://www.w3schools.com/tags/tag_title.asp) within the html header - https://github.com/ImperialCollegeLondon/drunc_ui/blob/33b8433fabd19ccf6ae7e2f05c2f674463b5dbc0/main/templates/main/base.html#L6-L10. This effectively sets the name of the tab in the browser.

**Pull request merged**